BackEnd/src/routes/api/course.js

```diff
@@ -15,15 +15,15 @@ createRoutes([
                 args = extractArguments(req.body, 
                 [
                     { key: 'year', type: 'number' },
-                    { key: 'subjectId', type: 'string' },
+                    { key: 'subjectId', type: 'number' },
                 ]);
             }
             catch (err) 
             {
                 return res.status(400).json( {error: err.message} );
             }
 
-            const id = args.year + args.subjectId + Math.ceil(Math.random()*3);
+            const id = Math.ceil(Math.random()*30);
```

Comment:
Ne random id zwischen 1 & 30? Was passiert wenn ich zwei mal die gleiche Id würfel?

Reply:
Dann hat man eben zwei mal die gleiche Fake-Id. Fand ich besser, als bei jeder Abfrage die gleiche Id zurückzusenden. Feature wird in #57 umgesetzt, dann gibt es auch die richtige Id.